pkg/sql/distsqlrun/algebraic_set_op.go

```diff
@@ -37,9 +35,20 @@ type algebraicSetOp struct {
 	opType                  AlgebraicSetOpSpec_SetOpType
 	ordering                Ordering
 	types                   []sqlbase.ColumnType
-	datumAlloc              *sqlbase.DatumAlloc
+	datumAlloc              sqlbase.DatumAlloc
+
+	leftAccum, rightAccum streamGroupAccumulator
+	leftGroup, rightGroup []sqlbase.EncDatumRow
+	allCols               columns
+	rightMap              map[string]int
+	leftIdx               int
```

Comment:
Could you document some of these fields? At least the ones that don't have obvious meanings (to me anyway) like cmp, leftIdx, rightMap.